src/interfaces/index.ts

```diff
@@ -487,6 +487,7 @@ export interface SalesOfferPackageWithErrors extends SalesOfferPackage {
 export interface TicketPeriod {
     startDate?: string;
     endDate?: string;
+    dateInput: ProductDateInformation;
```

Comment:
As it stands this entire object is passed into the matching json so we'll end up with the `dateInput` in there as well, we should ideally make it so we only pass the `startDate` and `endDate` in `userData.ts`

Reply:
I've updated now with new type 'TicketPeriodWithInput'